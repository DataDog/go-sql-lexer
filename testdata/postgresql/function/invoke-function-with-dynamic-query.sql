SELECT * FROM dynamic_query('SELECT * FROM users WHERE id = 1') AS t(id integer, name text, email text);
