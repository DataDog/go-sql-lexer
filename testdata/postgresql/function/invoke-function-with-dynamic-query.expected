[
    {
      "expected": "SELECT * FROM dynamic_query ( ? ) AS t ( id integer, name text, email text )"
    }
  ]
  